{"Name":"default_namespace","Result":false,"SerializedDocument":"document\r\ndefault <https://default.org/>\r\nprefix ex <https://example.org/>\r\nentity(ex:e)\r\nendDocument","DeserializedDocument":null,"Exception":"Traceback (most recent call last):\r\n  File \"C:\\Users\\Mat˙Ü\\bakalarka\\python-prov\\src\\main.py\", line 38, in <module>\r\n    class_.deserialize(sys.argv[2])\r\n  File \"C:\\Users\\Mat˙Ü\\bakalarka\\python-prov\\src\\test_case.py\", line 21, in deserialize\r\n    document = document.deserialize(fr\"..\\data\\{self.filename}.{format_type}\", format=format_type)\r\n  File \"C:\\Users\\Mat˙Ü\\AppData\\Local\\Programs\\Python\\Python39\\lib\\site-packages\\prov\\model.py\", line 2536, in deserialize\r\n    return serializer.deserialize(f, **args)\r\n  File \"C:\\Users\\Mat˙Ü\\AppData\\Local\\Programs\\Python\\Python39\\lib\\site-packages\\prov\\serializers\\provn.py\", line 25, in deserialize\r\n    raise NotImplementedError\r\nNotImplementedError","Description":"PROVN deserialization in python is not implemented yet.\r\nIt has been worked on but abandoned as you can see in this issue on libraries github https://github.com/trungdong/prov/issues/122\r\n"}